src/concrete/ml/sklearn/base.py

```diff
@@ -645,17 +645,19 @@ def predict(self, X: Data, fhe: Union[FheMode, str] = FheMode.DISABLE) -> numpy.
 
                 # If the inference should be executed using simulation
                 if fhe == "simulate":
+                    is_crt_encoding = self.fhe_circuit.statistics["packing_key_switch_count"] != 0
 
-                    # If the old simulation method should be used
-                    if USE_OLD_VL:
+                    # Whether the old simulation method should be used based on USE_OLD_VL
```

Comment:
I would rephrase this with : 
```
# If the virtual library method should be used 
# For now, use the virtual library when simulating circuits that uses CRT 
# decomposition because the official simulation is too slow 
# FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/4391
```

Reply:
in what?

Reply:
in this : https://github.com/zama-ai/concrete-ml/pull/619/files#r1568710561